### PR TITLE
Feature/skare 20 class game handle raise

### PR DESCRIPTION
**Pull request by robcas12**
This is my Class Game. This was the task that SKARE-20, "Handle raise function." I ended up making a few features within the Game class, and it manages player turns/betting actions. It integrates with the Pot class for bet tracking and management. The next step I am considering is improving the user interface for better game interaction. I want to optimize for more complex scenarios and larger player numbers. This is just the bare bones I need to improve these features, so any feedback is appreciated :).

### CODE REVIEW

**Review by mccueki — APPROVED** [review]
Great work! I just reveiwed all the changes and its looking good so far. This should implement well.

File: source/CardGames.py [hunk 1]
```diff
@@ -155,4 +232,40 @@ def dealCards(self, numCards: int, players: "list[Player]"):
 
   def resetDeck(self):
     self.deck.reset()
-    self.deck.shuffle()
\ No newline at end of file
+    self.deck.shuffle()
+
+
+# has_pair function---------------------------------------------------------------
+
```
[the remark this answers is not in the document]
Comment (robcas12) [reply source/CardGames.py:239]: Got it will do that. Thanks.

[the remark this answers is not in the document]
Comment (robcas12) [reply source/CardGames.py:239]: Got it just made the indentation adjustments so that the functions fit within the dealer class. Thanks.

**Comment by robcas12** [comment]
> Great work! I just reveiwed all the changes and its looking good so far. This should implement well.

Thanks.

**Review by erinyooey — APPROVED** [review]

**Pull request merged**